File: Write Number in Expanded Form.dart
String expandedForm(int num) {
  List<String> result = [];
  List<String> sub_a = num.toString().split("");
  for(int i =0;i<num.toString().length;i++){
    String num_s = sub_a.sublist(i,i+1).join("");
    if(num_s!="0"){
      result.add(num_s+"0"*sub_a.sublist(i+1,).join("").length); 
    }
  }
  return result.join(" + ");
}

File: Write Number in Expanded Form/Write Number in Expanded Form.py
import re
def expanded_form(num):
    num_s = str(num)
    arr = [num_s[i]+"0"*(len(num_s[i+1:])) for i in range(len(num_s))]
    arr_f = list(filter(lambda x:not(re.match("^0+$",x)),arr))
    return " + ".join(arr_f)
